### ISSUE TITLE
Auto categorize manga based on tracking status

### ISSUE THREAD

**Issue by KaBankz** [Aidoku/Aidoku#141]
Instead of making categories "Reading", "Plan to read", etc... and manually moving manga to and from these categories, this could be automated.

Every manga that has tracking would be grouped together, so manga marked as reading get grouped together into "Reading" and so on.

This would be different from normal categories, and can not work at the same time, it would have to be toggled for either auto categorize, or manually categorize.

Auto categorize syncs with tracking, so if tracking status changes, it would move the manga to that new category. This would not allow the user to move around manga on a whim because it would no longer be in sync with tracking, so if a user wants to move manga they have to change its tracking status. Tachi does this and it's amazing.

Of course, people who want to manually categorize their manga, separate from tracking, can disable auto categorization and manually create and manage categories like the current implementation.

***tldr**: automatically group manga based on their tracking status*

**Comment by okarin001** [comment]
If I remember correctly, Tachiyomi have this feature. It would be a great addition to Aidoku. I was thinking the same.
